src/utils/ledger-bridge-provider/BitcoinLedgerBridgeApp.js

```diff
@@ -1,46 +1,45 @@
 import { LedgerBridgeApp } from './LedgerBridgeApp'
-import {
-  ExecutionMode
-} from '@liquality/hw-web-bridge'
+import { ExecutionMode } from '@liquality/hw-web-bridge'
 
 export class BitcoinLedgerBridgeApp extends LedgerBridgeApp {
   async signMessageNew (...payload) {
-    return await this.callToBridge({
+    return this.callToBridge({
       action: 'signMessageNew',
       execMode: ExecutionMode.Async,
       payload
     })
   }
 
   async getWalletPublicKey (...payload) {
-    return await this.callToBridge({
+    return this.callToBridge({
       action: 'getWalletPublicKey',
       execMode: ExecutionMode.Async,
       payload
     })
   }
 
   async splitTransaction (...payload) {
-    return await this.callToBridge({
+    return this.callToBridge({
       action: 'splitTransaction',
       execMode: ExecutionMode.Sync,
       payload
     })
   }
 
   async createPaymentTransactionNew (...payload) {
-    return await this.callToBridge({
+    return this.callToBridge({
       action: 'createPaymentTransactionNew',
       execMode: ExecutionMode.Async,
       payload
     })
   }
 
   async serializeTransactionOutputs (...payload) {
-    return await this.callToBridge({
+    const result = await this.callToBridge({
```

Comment:
for consistency, can we also do `return this.callToBridge({` here?

Reply:
fixed, thanks!